conformance/fhir-ig-davinci-epdx/CHANGELOG

```diff
@@ -0,0 +1,7 @@
+Version 1
+- Changed the precision of json/profiles/fhir-ig-davinci-epdx/Patient-1.json's 
+Text '2020-07-07T13:26:22.0314215+00:00' could not be parsed at index 26 [Patient.meta.lastUpdated] 
+
+- Modified the expression for provenance-1
+From ($this.agent.who.resolve().is Practitioner or Device) implies exists()
```

Comment:
did we open a JIRA issue for this one?

Reply:
OK, I found it at https://jira.hl7.org/browse/FHIR-28451, recommend we include that here.
```suggestion
- https://jira.hl7.org/browse/FHIR-28451
From ($this.agent.who.resolve().is Practitioner or Device) implies exists()
```

Reply:
Was this accidentally omitted or you disagreed about adding a link to the issue?

Reply:
No,  it's not intentional, I had tried adding it as a commit, and may have overwritten with the folder change

Reply:
Fixed